content/react/concepts/hooks/context/context.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: 'React Context'
```

Comment:
I see in this intro that context is a built-in API: https://beta.reactjs.org/reference/react/apis

Perhaps we can create a new concept that is on the same level as hooks. Rather than nesting the `context` folder in `hooks`, let's move it one level up to lie underneath `concepts`. Perhaps we can name it `api` instead and add "Context" as a heading within that file.

Reply:
@yangc95 I think moving the context folder outside to the concepts folder would suffice. Adding an additional `/api` folder without adding more entries like `useMemo` or `useCallback` seems a bit overdone.

Reply:
@DerTimonius Sure! We can do this for now and revisit when the time to add those other api terms is needed 🙂